hooks/useExamFiltering.ts

```
@@ -1,33 +1,99 @@
+              case "wi_ba_dual":
+              case "pi_ma":
+              case "ti_ma":
+              case "mi_ma":
+              case "wi_ma":
+              case "is_ma":
+                console.log(entry[course.key])
+                return entry[course.key] === selectedSemester;
+              default:
+                return false;
+            }
+          });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)